<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>597</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lucida Sans Unicode</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>КПО Лаб. 2, Ковынев М.В. 6304</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/logo.jpg</normaloff>:/img/logo.jpg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>500</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Настройка</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Добавляемая фигура</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="enable_square">
              <property name="font">
               <font>
                <family>Lucida Sans Unicode</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Квадрат с волнистыми сторонами</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="enable_SinWave">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Lucida Sans Unicode</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Отрезок синусоиды</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="enable_text">
              <property name="font">
               <font>
                <family>Lucida Sans Unicode</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Текст</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="enable_text_with_square">
              <property name="font">
               <font>
                <family>Lucida Sans Unicode</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Текст в квадрате</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="add_scene">
                <property name="text">
                 <string>Добавить сцену</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="clear_current_scene">
                <property name="text">
                 <string>Очистить текущую сцену</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_figure_size">
                <property name="maximumSize">
                 <size>
                  <width>400</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Lucida Sans Unicode</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Размер фигуры:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="figure_size_slider">
                <property name="maximumSize">
                 <size>
                  <width>500</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>40</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="pageStep">
                 <number>15</number>
                </property>
                <property name="value">
                 <number>70</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>DejaVu Sans</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Частота синуса:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sin_freq">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>200</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>DejaVu Sans</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Амплитуда синуса:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sin_ampl">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>20</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="maximumSize">
                 <size>
                  <width>1000</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Lucida Sans Unicode</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Цвет</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="random_color">
                <property name="text">
                 <string>Произвольный цвет</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="iterate_container">
              <property name="text">
               <string>Обход контейнера</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="clear_all">
              <property name="text">
               <string>Очистить все</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QMdiArea" name="mdi_area"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="load_from_file"/>
    <addaction name="save_to_file"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="load_from_file">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="save_to_file">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="clear">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Новая сцена</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Очистить все сцены</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Обойти контейнер</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>add_scene</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_current_scene</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_clear_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterate_container</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_3_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_all</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_action_2_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
